modal_utils/decorator_utils.py

```diff
@@ -1,7 +1,19 @@
 # Copyright Modal Labs 2022
+import datetime
 import functools
 import inspect
 
+from modal.exception import deprecation_warning
```

Comment:
i think we generally want to avoid cyclical dependencies between packages, but it's probably fine to have this for a short while

Reply:
Ah, I didn't even realize this was in a separate package :O A good motiation to get rid of the deprecated stuff soon!